### DISCUSSION TITLE
PostGIS synchronization after pushed files with QFieldCloud

### DISCUSSION THREAD

**Discussion by ronanren** [opengisch/QField#2467] [Q&A]
Dear QField developper team

First of all, thanks for all your great work and I have question about functionnality of PostGIS synchronization after pushed files with QFieldCloud.
I tried on your server app.qfield.cloud and also on my ubuntu server which I host QFieldCloud.
When I push files on QFieldCloud server after editing on QField, synchronization on QFieldCloud server works well but PostGIS database is not updated.

There's a solution or setting to set up to make it work or the solution isn't yet developped ?

Best regards,
Ronan

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Reply by Phertl** [reply]
Hi Ronan,

Is the Postgis DB set to publicity? Adding new features from different computers works fine correct?
I had an issue because the ID Field was not unique.

Best regards
Paul

**Reply by ronanren** [reply]
Yes the Postgis DB is public and id was unique, the synchronization from QFieldCloud to Postgis works for you ?

**Reply by Phertl** [reply]
Yes the synchronization from QFieldCloud to Postgis works for me. My database is hosted on AWS. I created a "testorganization" with 2 users. Everything works really good.

**Reply by aliasatp** [reply]
Have you got a "offiline" or "online" setting? In my project the sync action from qfiledcloud updates the gpkg dataset but not the native POSTGIS one. My setting is offline.

**Reply by ronanren** [reply]
Thanks for your reply, the development team assured me that the sync with the native postgis database should work, I hope to find a solution to make this feature work

**Comment by suricactus** [comment]
Hey, @ronanren . Thanks for testing QFieldCloud. Can you share a little bit more details about your project? How you created it and how you configured your project. 

Also, check my answer here https://github.com/opengisch/QField/discussions/2508#discussioncomment-2322119 and let me know if it answers your question.
